demo/kserve/custom-manifests/caikit/caikit-tgis-isvc-http.yaml

```diff
@@ -1,20 +0,0 @@
-apiVersion: serving.kserve.io/v1beta1
```

Comment:
This duplicates https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/custom-manifests/caikit/caikit-tgis/caikit-tgis-isvc.yaml